recipes-demos/qmltermwidget/qmltermwidget.bb

```diff
@@ -0,0 +1,21 @@
+# Recipe taken from https://github.com/webOS-ports/meta-webos-ports
+# Commit: 528204c26b1550f54b7b93cbc9c32352ab5b0839
+SUMMARY = "A terminal emulator QML widget, based on LXQt's QTermWidget"
+LICENSE = "GPL-2.0-only"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=4641e94ec96f98fabc56ff9cc48be14b"
+
+PV = "0.14.1+git"
+SRCREV = "59f967d5e1f6e9ce8e1632d9405422b071d93d30"
```

Comment:
@paresh-bhagat12 Quick query - Any particular reasons for using the above commit for qmltermwidget and not the latest tip - 63228027e1f97c24abb907550b22ee91836929c5 (which again seems to be from 2022)

Reply:
@cshilwant  I used a commit before it was moved from QT5 to QT6